### PR TITLE
PMM-6985 add cluster deletion force mode

### PR DESCRIPTION
**Pull request by lunaticusgreen**
[PMM-6985](https://jira.percona.com/browse/PMM-6985)

https://github.com/Percona-Lab/pmm-submodules/pull/1256

### CODE REVIEW

File: pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.service.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ export const KubernetesService = {
   getKubernetes() {
     return apiRequestManagement.post<KubernetesListAPI, any>('/DBaaS/Kubernetes/List', {});
   },
-  deleteKubernetes(kubernetes: Kubernetes) {
-    return apiRequestManagement.post<any, any>('/DBaaS/Kubernetes/Unregister', toAPI(kubernetes));
+  deleteKubernetes(kubernetes: Kubernetes, force) {
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.service.ts:13]:
```suggestion
  deleteKubernetes(kubernetes: Kubernetes, force: boolean) {
```

File: pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.service.ts [hunk 2]
```diff
@@ -21,8 +21,9 @@ export const KubernetesService = {
   },
 };
 
-const toAPI = (kubernetes: Kubernetes) => ({
+const toAPI = (kubernetes: Kubernetes, force) => ({
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.service.ts:24]:
```suggestion
const toAPI = (kubernetes: Kubernetes, force: boolean) => ({
```

File: pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.hooks.ts [hunk 3]
```diff
@@ -30,10 +30,10 @@ export const useKubernetes = (): [Kubernetes[], DeleteKubernetesAction, AddKuber
     }
   };
 
-  const deleteKubernetes = async (kubernetesToDelete: Kubernetes) => {
+  const deleteKubernetes = async (kubernetesToDelete: Kubernetes, force) => {
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.hooks.ts:33]:
```suggestion
  const deleteKubernetes = async (kubernetesToDelete: Kubernetes, force: boolean) => {
```

Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.hooks.ts:33]:
```suggestion
  const deleteKubernetes = async (kubernetesToDelete: Kubernetes, force: boolean) => {
```

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx [hunk 4]
```diff
@@ -107,37 +107,48 @@ export const KubernetesInventory: FC<KubernetesProps> = ({
...
+          onSubmit={() => {}}
+          render={({ form, handleSubmit }) => (
+            <form onSubmit={handleSubmit}>
+              <>
+                <h4 className={styles.deleteModalContent}>
+                  {Messages.kubernetes.deleteModal.confirmMessage}
+                </h4>
+                <FormElement
+                  dataQa="form-field-force"
+                  label="Force mode"
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx:120]: Move to Messages file.

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.service.ts (hunk 2 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.service.ts]: done

File: pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.service.ts (hunk 1 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.service.ts]: done

File: pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.hooks.ts (hunk 3 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/Kubernetes.hooks.ts]: done

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx (hunk 4 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx]: done

File: pmm-app/src/pmm-dbaas/DBaaS.styles.ts [hunk 5]
```diff
@@ -1,8 +1,12 @@
 import { CSSProperties } from 'react';
+import { css } from 'emotion';
 
 export const styles = {
   disabled: {
     opacity: 0.6,
     pointerEvents: 'none',
   } as CSSProperties,
+  panelContentWrapper: css`
+    margin-top: 20px;
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/DBaaS.styles.ts:10]: Change this object to a getStyles function and use theme values.

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: pmm-app/src/pmm-dbaas/DBaaS.styles.ts (hunk 5 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/DBaaS.styles.ts:10]: https://jira.percona.com/browse/PMM-7027 yes, we already have a ticket for it so it will be done there

**Pull request merged**